Dependencies/xr/Include/XR.h

```diff
@@ -131,11 +140,11 @@ namespace xr
                 std::unique_ptr<Impl> m_impl{};
             };
 
-            Session(System& system, void* graphicsDevice);
+            static arcana::task<std::shared_ptr<Session>, std::exception_ptr> CreateAsync(System& system, void* graphicsDevice);
             ~Session();
 
-            Session(Session&) = delete;
-            Session& operator=(Session&&) = delete;
+            // Do not use, call CreateAsync instead.
+            Session(System& system, void* graphicsDevice);
```

Comment:
Can this not be private?

Reply:
Fixed by using shared_ptr directly instead of make_shared.